### DISCUSSION TITLE
Actividad Docker - GRUPO 1

### DISCUSSION THREAD

**Discussion by FranciscoMontiron** [frlp-utn-ingsoft/catedra#109] [Show and tell]
https://github.com/Fedebravo12/vetsoft

**Comment by rodriguezemautn** [comment]
Correcciones: 
- el archivo Dockerfile se escribe con inicial mayuscula
-el archivo env-example no lleva . Delante
- las variables declaradas en el .env no se consumen
- README.md no sugiere como se accede a la app
- el comando docker run sugerido con esta en detach (-d) al cerrar la consola se detiene el contenedor (Prestar atencion a las buenas practicas)
- los endpoint dan error, no se migra la base de datos(verificar orden del CMD en el Dockerfile o utilizar un script.sh en un ENTRYPOINT) 

Puntos aprobados:
* construye la imagen
* buen tamaño de imagen

Salida de comandos:
$ docker ps  -a
CONTAINER ID   IMAGE              COMMAND                  CREATED         STATUS         PORTS                                       NAMES
bfd386654421   vetsoft-app:v1.0   "python manage.py ru…"   3 minutes ago   Up 3 minutes   0.0.0.0:4000->8000/tcp, :::4000->8000/tcp   vetsoft-container
$ docker images -a
REPOSITORY    TAG       IMAGE ID       CREATED         SIZE
vetsoft-app   v1.0      f153082345f9   4 minutes ago   307MB

**Reply by germansalinas1994** [reply]
Ema, ya lo subimos corregido hace unos días acá:
https://github.com/orgs/frlp-utn-ingsoft/discussions/120

Estamos atentos en caso de que falte algo
